.pylintrc

```diff
@@ -0,0 +1,2 @@
+[MASTER]
+disable=fixme,too-few-public-methods,too-many-instance-attributes,too-many-arguments,logging-fstring-interpolation,too-many-locals,duplicate-code,pointless-string-statement,bare-except,invalid-name,import-error,broad-except
```

Comment:
Had borrowed some from your solver-template pylint. 
These are the ignores now:
fixme,too-few-public-methods,too-many-arguments,too-many-locals,pointless-string-statement,import-error

Reply:
Among the remaining, I would think that 

- `too-many-locals` indicates you have code blocks that have become too large and are likely not focused on accomplishing a single task, thus many variables are being created within the function. I would try to resolve this.
- `pointless-string-statement` also looks like code that could just be removed.
- `import-error` should probably also not be silenced.

Reply:
how do you ignore the secret key file, import-error was ignored since keys file isn't pushed to github:
https://github.com/cowprotocol/ebbo/actions/runs/4683373799/jobs/8298384068?pr=10

Reply:
I would think that if the project is relying on a file that is not comitted to the project, then it sounds like you should be putting those "secrets" into a dotenv file (i.e. a file `.env`) that contains a list of constants like 

```
DUNE_API_KEY=
INFRUA_KEY=
```

Then you can `load_dotenv()` and read the environment variables you need in this way rather than an "import" statement.

Here is some docs (its very easy to use) https://pypi.org/project/python-dotenv/

```
pip install python-dotenv
```

and maybe something like this is the flavour you are looking for:

```py
from dotenv import dotenv_values

config = dotenv_values(".env")  # config = {"USER": "foo", "EMAIL": "foo@example.org"}
```